element/index.js

```diff
@@ -65,3 +66,20 @@ export function renderToString( element ) {
 
 	return renderToStaticMarkup( element );
 }
+
+export function concatValues( value1, value2 ) {
+	const toArray = value => Array.isArray( value ) ? Children.toArray( value ) : [ value ];
+
+	return toArray( value1 )
```

Comment:
no, `Children.toArray` is not enough, and I figured out, I'd reset the keys to avoid any confusion/duplication